Go 1.15: conversion from int to string yields a string of one rune

Go 1.15 rc 1 on Fedora Rawhide:

```
Testing    in: /builddir/build/BUILD/goquery-1.5.1/_build/src
         PATH: /builddir/build/BUILD/goquery-1.5.1/_build/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin
       GOPATH: /builddir/build/BUILD/goquery-1.5.1/_build:/usr/share/gocode
  GO111MODULE: off
      command: go test -buildmode pie -compiler gc -ldflags "-X github.com/PuerkitoBio/goquery/version=1.5.1 -extldflags '-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '"
      testing: github.com/PuerkitoBio/goquery
github.com/PuerkitoBio/goquery
# github.com/PuerkitoBio/goquery
./bench_iteration_test.go:34:10: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
FAIL	github.com/PuerkitoBio/goquery [build failed]
```

Please see https://github.com/golang/go/issues/32479

--- a/bench_iteration_test.go
+++ b/bench_iteration_test.go
@@ -1,6 +1,7 @@
 package goquery
 
 import (
+	"fmt"
 	"testing"
 )
 
@@ -31,7 +32,7 @@ func BenchmarkMap(b *testing.B) {
 	sel := DocW().Find("td")
 	f := func(i int, s *Selection) string {
 		tmp++
-		return string(tmp)
+		return fmt.Sprintf("%d", tmp)
 	}
 	b.StartTimer()
 	for i := 0; i < b.N; i++ {

